src/renderers/shared/fiber/ReactFiberErrorLogger.js

```diff
@@ -99,12 +99,15 @@ function logCapturedError(capturedError: CapturedError): void {
         'See https://fb.me/react-error-boundaries for more information.';
     }
 
-    console.error(
-      `${componentNameMessage} You should fix this error in your code. ${errorBoundaryMessage}\n\n` +
-        `${errorSummary}\n\n` +
-        `The error is located at: ${componentStack}\n\n` +
-        `The error was thrown at: ${formattedCallStack}`,
-    );
+    let combinedMessage = `${componentNameMessage} You should fix this error in your code. ${errorBoundaryMessage}\n\n`;
+    if (!__DEV__) {
+      combinedMessage += `${errorSummary}\n\n`;
+    }
+    combinedMessage +=
+      `The error is located at: ${componentStack}\n\n` +
+      `The error was thrown at: ${formattedCallStack}`;
+
+    console.error(combinedMessage);
   } else {
     console.error(
       `React caught an error thrown by one of your components.\n\n${error.stack}`,
```

Comment:
In production we use a try-catch, so the browser doesn't log it. Only our error logger.